.github/workflows/static-analysis.yaml

```diff
@@ -0,0 +1,25 @@
+name: Code style
+
+on:
+    pull_request:
+    push:
+        branches: [ master ]
+
+jobs:
+    php-cs-fixer:
+        name: PHP-CS-Fixer
```

Comment:
This isn't correct. We are running `phpcs` to check code style, not fixing it. PHP-CS-Fixer is a different tool entirely.